index.html

```diff
@@ -128,89 +128,9 @@ <h3>CC Ignite</h3>
         <h1>Our Staff</h1>
         <hr>
         <div class="featured-elem">
-            <div class="staff-card">
-                <div class="staf-img">
-                    <img src="img/Florentino_perez.jpg" alt="Academy_Staff">
-                </div>
-                <div class="staff-intro">
-                    <h3>Florentino Pérez</h3>
-                    <h4>Director</h5>
-                        <hr>
-                        <p>75 years old <br>Spanish businessman, civil engineer, former politician, and the current
-                            president of Real Madrid as well as Chairman and CEO.</p>
-                </div>
-            </div>
-
-
-            <div class="staff-card">
-                <div class="staf-img">
-                    <img src="img/jose-mourinho-1.jpeg" alt="Academy_Staff">
-                </div>
-                <div class="staff-intro">
-                    <h3>José Mourinho</h3>
-                    <h4>Coach</h5>
-                        <hr>
-                        <p>59 years old <br>Once dubbed "The Special One" by the British media,
-                            Mourinho is widely considered to be among the greatest.</p>
-                </div>
-            </div>
-
-
-            <div class="staff-card">
-                <div class="staf-img">
-                    <img src="img/pep-guardiola.jpeg" alt="Academy_Staff">
-                </div>
-                <div class="staff-intro">
-                    <h3>Pep Guardiola</h3>
-                    <h4>Assistant Coach</h5>
-                        <hr>
-                        <p>51 years old <br>Spanish professional football manager and former player, who is the current
-                            manager of Premier League club Manchester City.</p>
-                </div>
-            </div>
 
+            <!-- Add dynamicly the speakers in the Js file -->
```

Comment:
- [ ] I think it would be great if you fix the styles here my pc size on 100% but it looks like this I will attach an image but since in the requirements it should be responsive so it would be great if you design displayed well on all the devices you can make the min-width on the media query more for example. ✅✨🙏🏻🔥🔥

- Your design
![image](https://user-images.githubusercontent.com/58553711/210464761-43a3030d-55c0-4b74-96a7-0a820f5df348.png)

- Given
![image](https://user-images.githubusercontent.com/58553711/210464893-f6be38fe-9808-4b7e-a215-604b12e59b77.png)